### ISSUE TITLE
Still changing external links

### ISSUE THREAD

**Issue by cramerhwood** [WPGlobus/WPGlobus#27]
I am having this same issue with a site. I think it is do to the use of sub domains. So the domains are structured as such:

- 'blog.mydomain.com' with external links pointing to 'mydomain.com'
- 'mydomain.com/about-us' becomes 'mydomain.com/uk/about-us'

I'm using 1.4.8

Any suggestions?

**Comment by tivnet** [comment]
Right, there is a piece of code that removes any prefix from the domain (like 'www', 'dev', and so on).
Apparently, it removed 'blog'.
Where do you have those links? In the menu?

**Comment by cramerhwood** [comment]
Yes sir. In the menu.



Sent from my Verizon Wireless 4G LTE smartphone


-------- Original message --------

**Comment by cramerhwood** [comment]
I apologize, I just saw this. Yes, the links are in the main menu.

Aaron Page | President

p: 214.453.0081
c:  832.622.2831
e:  apage@page.agency<mailto:apage@page.agency>
w: page.agency<http://t.sidekickopen50.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs4WJ4X8W1qg0Yb1p8hNHW2z8P1C56dPHYf1xBCzH02?t=http%3a%2f%2fpage.agency%2f%3futm_source%3dAaron%26utm_medium%3dEmail%26utm_campaign%3dOutreach&si=5144745654878208&pi=ab27432e-5910-4759-8654-a8c487b00931>

[The Page Agency]<http://t.sidekickopen50.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs4WJ4X8W1qg0Yb1p8hNHW2z8P1C56dPHYf1xBCzH02?t=http%3a%2f%2fpageagency.net%2f&si=5144745654878208&pi=ab27432e-5910-4759-8654-a8c487b00931>

( Formerly BigDigi LLC )


CONFIDENTIALITY NOTICE: This facsimile/e-mail message is intended only for the person or entity to which it is addressed and may contain CONFIDENTIAL or PRIVILEGED material. Any unauthorized review, use, disclosure or distribution is prohibited. If you are not the intended recipient, please contact the sender and destroy all copies of the original message. If you are the intended recipient but do not wish to receive communications through this medium, please advise the sender immediately.

**Comment by ipresentyou** [comment]
Any suggestions for this?

**Comment by zEdmundo** [comment]
Same problem here.
